README.md

```diff
@@ -91,6 +91,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Present Technologies](https://www.present-technologies.com/) [:rocket:](https://www.present-technologies.com/careers/) | Delivering Quality Software. | `Coimbra` `Porto` <br> `Sertã` |
 | [Radically Digital](https://radically.digital/) [:rocket:](https://radically.digital/careers/) | Software, ideation, strategy and design consultancy. | `Lisboa` |
 | [Reaktor](https://www.reaktor.com) [:rocket:](https://www.reaktor.com/careers) | A global strategy, design, and technology partner. | `Lisboa` |
+| [Servinform](https://www.servinform.es/) [:rocket:](https://www.servinform.es/unete-al-equipo/)| Consultancy and Automation. | `Lisboa` |
```

Comment:
```suggestion
| [Servinform](https://www.servinform.es/pt-pt/) [:rocket:](https://www.servinform.es/pt-pt/junte-se-a-equipa/)| Consultancy and Automation. | `Lisboa` |
```

Probably this is first company using Spanish as the default language, so I'm bit unsure about the defaults.

I think using the URLs for Portuguese or English is preferable.

Reply:
Agreed!